sgl-kernel/amd/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.18)
```

Comment:
Please remove this; we only use CMakeLists.txt for clangd indexing, so it's not necessary.